docker_ci_deploy/__main__.py

```diff
@@ -300,6 +327,19 @@ def docker_push(self, tag):
         self._docker_cmd(['push', tag])
 
 
+def _git_rev_parse(ref, *options):
+    out, _ = cmd(['git', 'rev-parse'] + list(options) + [ref], quiet=True)
```

Comment:
Guess I should try do the same kind of thing I did with the `docker` executable.